src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -85,6 +93,22 @@ public EmployeeDto updateEmployeeEntity(EmployeeDto employeeDto) {
     return newEmployeeDto;
   }
 
+  @Override
+  @Transactional(isolation = Isolation.SERIALIZABLE, propagation = Propagation.REQUIRED)
+  public List<Employee> bulkUpdate(BulkEmployeeDto bulkEmployeeDto) {
+    final Map<AcceptedFields, String> acceptedFieldsMap = bulkEmployeeDto
+        .getAcceptedFieldsMap();
+
+    return bulkEmployeeDto.getEmployeeIds().stream()
+        .map(employeeRepository::findById).map(employee -> {
+          acceptedFieldsMap.forEach((field, value) -> field.setValue(employee.get(), value));
+          return employee;
+        }).
+            map(employee -> employee.get())
```

Comment:
method reference could be used here...change it to employee::get or Employee::get